ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/UrlBar/TabLocationView.swift

```diff
@@ -243,6 +243,8 @@ class TabLocationView: UIView {
 
   private let placeholderLabel = UILabel().then {
     $0.text = Strings.tabToolbarSearchAddressPlaceholderText
+    $0.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+    $0.setContentCompressionResistancePriority(.defaultHigh, for: .horizontal)
```

Comment:
`defaultHigh` should be the default compression resistance

Reply:
Removed.